app/src/main/java/org/covidwatch/android/ui/home/HomeViewModel.kt

```diff
@@ -26,11 +26,11 @@ class HomeViewModel(
     private val _warningBannerState = MutableLiveData<WarningBannerState>()
     val warningBannerState: LiveData<WarningBannerState> get() = _warningBannerState
 
-    private val _userFlow = MutableLiveData<UserFlow>()
-    val userFlow: LiveData<UserFlow> get() = _userFlow
+    private val _navigateToOnboardingEvent = MutableLiveData<Event<Unit>>()
+    val navigateToOnboardingEvent: LiveData<Event<Unit>> get() = _navigateToOnboardingEvent
 
-    private val _isRefreshing = MediatorLiveData<Boolean>()
-    val isRefreshing: LiveData<Boolean> get() = _isRefreshing
+    private val _exposureSummary = MutableLiveData<ExposureSummary>()
+    val exposureSummary: LiveData<ExposureSummary> get() = _exposureSummary
```

Comment:
Use `PreferenceStorage` for this. Check `ExposureNotificationViewModel` for an example. LiveData from the preferences get updates internally so we have latest values all the time.

Plus it uses `CovidExposureSummary`, a UI class for `ExposureSummary` class from the EN lib.